homeassistant/components/husqvarna_automower/services.yaml

```diff
@@ -19,3 +19,19 @@ override_schedule:
           options:
             - "mow"
             - "park"
+
+override_schedule_workarea:
+  target:
+    entity:
+      integration: "husqvarna_automower"
+      domain: "lawn_mower"
+  fields:
+    duration:
+      required: true
+      example: "{'days': 1, 'hours': 12, 'minutes': 30}"
+      selector:
+        duration:
+          enable_day: true
+    work_area_id:
+      required: true
+      example: "123"
```

Comment:
Currently the work_area_id is not exposed to the user in HA, so it'snot possible to enter it. Here are three solutions, I'm not sure, what would be beste:
1. Generate a service, which lists the work area ids and the human readable names: eg: `123: Front lawn`
2. Add extra_state_attributes to the work_areas_sensor: https://github.com/home-assistant/core/blob/f934fea754a8c27fe456648b27d80d8dad5d305f/homeassistant/components/husqvarna_automower/sensor.py#L338-L345 , which show the work area ids and the human readable names: eg: `123: Front lawn`
3. Request to enter der human readable name here, and then search for the work_area_id. Problem: The names might not be unique.

Reply:
Option 2 is fine in this case. Please exclude the list from the recorder